README.md

```diff
@@ -1 +1,226 @@
+```
+
+Create a featured branch besides main branch :
+
+```sh
+   git checkout -b nameofyournewbranch
+```
+
+### Install <br/>
+
+Install this project with:<br/>
+
+a) Enter to your repo's folder:
+
+```sh
+  cd helloMicroverse_project
+```
+
+b) For Webhint tool, install this command :
+
+```sh
+     npm install --save-dev hint@7.
+```
+
+c) For StyleLint tool, install this command:
+
+```sh
+   npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+```
+
+### Usage
+
+To run this project, execute the following command:
+
+a) For Webhint tool:
+
+```sh
+  npx hint .
+```
+b) For StyleLint tool:
+```sh
+    npx stylelint "**/*.{css,scss}"
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+   npx hint .
+   npx stylelint "**/*.{css,scss}"
+```
+
+
+### Deployment
+
+You can deploy this project using:
+- Github pages
+- Netlify 
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👩🏽‍💻🌸 **Author**
+
+- GitHub: [@IvonneBenitesRodriguez](https://github.com/IvonneBenitesRodriguez)
+- Twitter: [@IvonneBenitesR](https://twitter.com/IvonneBenitesR)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ivonnebenites/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ]  - I will add the a navbar with 3 categories: Home, Products and Contact.
+- [ ]  - I will add a section with a class. There I will add pictures and text.
+- [ ]  - I will add a footer section with social-network information. 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, and consider Linters configuration was useful to you,  you can give me a ⭐️ so I can be highly motivated by your support.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for giving me the opportunity to make real my dreams.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```

Comment:
- [ ] Hey Ivonne, I really appreciate how well you documented each section of the readme I just have a tiny suggestion on the "License" section here in the text below the MIT link which reads as "_NOTE: we recommend using the MIT license..." is a section description to guide the author on what type of license is advised but this should be removed from the final draft so I kindly suggest you remove it as it makes your project look unprofessional 👶
- This is the entire text you should remove 👇
  > 
_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._

Reply:
Hello @AbelG101,
Very nice to meet you 😊 
Thank you for your suggestion, I appreciate it.
 I modified and updated the part you suggested me.
Best regards,